packages/postgres-database/setup.py

```diff
@@ -62,8 +62,9 @@ def read_reqs(reqs_path: Path) -> set[str]:
     },
     entry_points={
         "console_scripts": [
-            "simcore-postgres-database=simcore_postgres_database.cli:main",
+            "simcore-service-postgres-database=simcore_postgres_database.cli:main",
```

Comment:
MINOR: make sure you keep this listing in alphabetical order. YOu can use vscode commands for that

Same with all listings in `console_scripts`

Reply:
thx